src/coreComponents/python/modules/geosx_xml_tools_package/geosx_xml_tools/unit_manager.py

```diff
@@ -46,54 +46,215 @@ def buildUnits(self):
         """Build the unit definitions."""
 
         # Long, short names for SI prefixes
-        prefixes = {'giga':  {'value': 1e9,  'alt': 'G'},
-                    'mega':  {'value': 1e6,  'alt': 'M'},
-                    'kilo':  {'value': 1e3,  'alt': 'k'},
-                    'hecto': {'value': 1e2,  'alt': 'H'},
-                    'deca':  {'value': 1e1,  'alt': 'D'},
-                    '':      {'value': 1.0,  'alt': ''},
-                    'deci':  {'value': 1e-1, 'alt': 'd'},
-                    'centi': {'value': 1e-2, 'alt': 'c'},
-                    'milli': {'value': 1e-3, 'alt': 'm'},
-                    'micro': {'value': 1e-6, 'alt': 'mu'},
-                    'nano':  {'value': 1e-9, 'alt': 'n'}}
+        prefixes = {
```

Comment:
Let's restore original formatting and wrap these into `# yapf: disable`?

Reply:
Agreed